.github/workflows/e2e.yml

```diff
@@ -11,13 +11,9 @@ jobs:
       fail-fast: false
       matrix:
         # Run all matrix env at once because we will not deploy demo app to MRT.
-        node: [16, 18, 20, 22]
+        node: [18, 20, 22]
```

Comment:
Just realized this, we removed Node 16, but still needs npm 8? npm 8 wont run on Node 16+ 🤔

Reply:
this is non-blocking comment

Reply:
Ok we can fix it later after the release. 

Cc @joeluong-sfcc

Reply:
npm 8 is compatible with Node 18. 
It might continue to work with node 20 and 22 but is not recommended.
While we don't see any failures or misbehaviors so far, it seems to be best to add exclusions for Node 20 + NPM 8 and Node 22 + NPM 8